docs/clients/promtail/configuration.md

```diff
@@ -979,43 +1017,47 @@ sync_period: "10s"
 
 ## Example Docker Config
 
+It's fairly difficult to tail docker files on a standalone machine because they are in different locations for every OS.  We recommend the [docker logging driver](../../../cmd/docker-driver/README.md) for local docker installs or docker-compose.
```

Comment:
```suggestion
It's fairly difficult to tail Docker files on a standalone machine because they are in different locations for every OS.  We recommend the [Docker logging driver](../../../cmd/docker-driver/README.md) for local Docker installs or Docker Compose.
```

Small nit.